unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Clipboard/Clipboard.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+
+public static class Clipboard
```

Comment:
I think we can add some test coverage to at least cover if the `ClipboardReadPromise` queue logic works as intended.